src/components/Dialogs/Multistepper/Multistepper.style.ts

```diff
@@ -0,0 +1,81 @@
+import styled from '@emotion/styled'
+import { Text, Button } from '@/shared/components'
+import { colors, sizes, breakpoints, typography } from '@/shared/theme'
+
+type CircleProps = {
+  isFilled?: boolean
+}
+
+type isActiveProp = {
+  isActive?: boolean
+}
+
+export const StyledHeader = styled.div`
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  min-height: 80px;
+  border-bottom: 1px solid ${colors.gray[500]};
+  margin: -${sizes(4)} -${sizes(4)} 0;
```

Comment:
I'm a little confused why these margins and paddings are set like this. Can you do it without negative values?

Reply:
The general dialog has this padding because  almost every dialog that will come in the future will have this padding as well, and only few of them will have borders inside. Either I could remove padding from the general dialog and add custom paddings for every dialog specifically or add padding to the general with negative margins only for those dialogs that have border inside. Went for the second approach - thought it's less work in a long run

Reply:
We probably need those margins but I'm a bit concerned with the coupling here, now we will need to remember to update those values if we update the base dialog. Maybe in the base dialog we could try specifying a responsive CSS var, something like `--dialog-padding` and use it both in dialog and here? Let me know your thoughts, I can also share more details on how this could work.

One thing though, do we need a vertical margin/padding as well? I think we could only use the horizontal ones to stretch the border 100%?